packages/fxa-settings/src/models/integrations/web-integration.ts

```diff
@@ -53,6 +53,12 @@ export class BaseIntegrationData extends ModelDataProvider {
   @bind(T.snakeCase)
   entrypointVariation: string | undefined;
 
+  // TODO - Validation - Needs @IsEncodedUrl()
+  // This is for SubPlat redirects which use the web integration
+  @IsOptional()
+  @bind(T.snakeCase)
```

Comment:
Is there a reason not to validate that it's a url?

Reply:
I didn't see `IsEncodedUrl` as an option from `class-validator` but I'll see if there's another option. I'll at least test it's a string yeah, thanks.